docs/content/v0.20/release.md

```diff
@@ -0,0 +1,60 @@
+- Because of the prohibition of self references, Git will not contain the exact bytes of our Helm chart definitions. Where a Helm chart states its own version or supplies a default value for a container image reference to an image built from the same release, the bytes in Git have a placeholder for that image's tag and the process of creating the published release artifacts fills in that placeholder. Think of this as being analogous to the linking done when building a binary executable file.
+
+## Technology
+
+There will be a GitHub workflow that creates the published artifacts for each Git tag whose name starts with "v". The rest of the tag name is required to be a semver release identifier. For each tag `v$version` the following published artifacts will be created.
+
+- The container image for the kubestellar-controller-manager (KCM), at `ghcr.io/kubestellar/kubestellar-controller-manager` (or should it be `ghcr.io/kubestellar/kubestellar/controller-manager` from the ks/ks repo? or just `ghcr.io/kubestellar/controller-manager`?). Image tag will be `$version`.
+- The Helm chart (for installing the KCM for a WDS), at `ghcr.io/kubestellar/kubestellar-controller-manager-chart` with version `$version` and Helm "appVersion" `$version`. The `values.yaml` of this chart version has a default value for the image reference to the KCM and that value is `ghcr.io/kubestellar/kubestellar-controller-manager:$version`.
+
+## Testing and Instructions
+
+The unit tests (of which we have almost none right now), integration tests (of which we have none yet), and end-to-end (E2E) tests in this repository are run in the context of a local copy of this repository and test that version of this repository --- not using any published release artifacts.
+
+The end-to-end tests include ones written in `bash`, and these are the only documentation telling a user how to use the present version of this repository. Again, these tests do not use any published artifacts from a release of this repo.
+
+We will introduce another category of tests, _release tests_. These test a given release, using the published artifacts of that release. These tests will be executables that a contributor invokes, passing the identifier of the desired release on the command line. We will create a GitHub workflow that can be explicitly triggered and will apply the release tests (in the branch that the workflow is directed at) to the latest release in that branch. We will also create a GitHub workflow that is simply told which release to test.
+
+We will maintain a document that lists releases that pass our quality bar. The latest of those is thus the latest stable release. This document is updated in `main` as quality evaluations come in.
+
+We will maintain an examples document that shows users how to exercise a few scenarios using the latest stable release.
```

Comment:
Is the idea to update the current examples doc for that ?

Reply:
I think the examples already point to a "release" (or artifact) , it's only the e2e tests that use the current code